microsoft/electionguard-python
# #693 Create Election, Export Encryption Package

### Issue

Fixes #693 

### Description
This allows admins to create an election, view active elections on their home page, view details of an election, and export the encryption package from the election.

### Testing
1. Create a key ceremony
2. Have guardians join the key ceremony to complete it
3. Create an election and specify the key ceremony from above
4. Click the download button in the top right
5. Export the election

Expected: the encryption package should be exported to the appropriate place.

Admin home:
![image](https://user-images.githubusercontent.com/1769905/180653086-12d15e9c-019c-44da-b71f-68b1573f487f.png)

Create election:
![image](https://user-images.githubusercontent.com/1769905/180653107-b6a7e070-61f1-4bc2-ac24-4ca9888e0d7c.png)

View election:
![image](https://user-images.githubusercontent.com/1769905/180653118-3b8349b6-40ae-49da-b265-4b4da060a88e.png)

Export encryption package:
![image](https://user-images.githubusercontent.com/1769905/180653138-db589cd2-909a-46da-a801-698d89aa6ea6.png)

Encryption Success
![image](https://user-images.githubusercontent.com/1769905/180653143-81890903-f801-4c63-a2f6-56befc345078.png)
